packs/st2/actions/kv_get.py

```diff
@@ -13,7 +13,7 @@ def run(self, key, decompress=False):
         _key = self.client.keys.get_by_name(key)
 
         if not _key:
-            return None
+            raise Exception("Key does not exist")
```

Comment:
I did this vs `return`, because frankly it's great that the task was successful, but I didn't get back any data. So, fail IMO. Not sure why return was default.